### PR TITLE
Update CI configuration to  match with the new default branch

### PR DESCRIPTION
**Pull request by erdemedeiros**
Part of https://github.com/Activiti/Activiti/issues/4168

### CODE REVIEW

**Review by gicappa — APPROVED** [review]
LGTM!

**Review by atchertchian — APPROVED** [review]

File: .github/workflows/prepare-prerelease.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 on:
   push:
-    branches: [ master ]
+    branches: [ develop ]
```
Comment (atchertchian) [inline .github/workflows/prepare-prerelease.yml]:
```suggestion
    branches:
      - develop
```

File: .github/workflows/prepare-release.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 on:
   push:
-    branches: [ master ]
+    branches: [ develop ]
```
Comment (atchertchian) [inline .github/workflows/prepare-release.yml]:
```suggestion
    branches:
      - develop
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,5 @@
 # activiti-scripts
 
-[![Build Status](https://travis-ci.com/Activiti/activiti-scripts.svg?branch=master)](https://travis-ci.com/Activiti/activiti-scripts)
```
Comment (atchertchian) [inline README.md:3]: This could be replaced by GH badges

Comment (erdemedeiros) [reply README.md:3]: Fixed

**Pull request merged**